plugins/languages/treesitter/treesitter-rainbow.nix

```diff
@@ -0,0 +1,66 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}:
+with lib; let
+  helpers = import ../../helpers.nix {inherit lib;};
+in {
+  options.plugins.treesitter-rainbow =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "treesitter-rainbow";
+
+      package = helpers.mkPackageOption "treesitter-rainbow" pkgs.vimPlugins.nvim-ts-rainbow2;
+
+      disable =
+        helpers.defaultNullOpts.mkNullable (types.listOf types.str) "[]"
+        "List of languages you want to disable the plugin for.";
+
+      query =
+        helpers.defaultNullOpts.mkStr "rainbow-parens"
+        "The query to use for finding delimiters.";
+
+      strategy =
+        helpers.defaultNullOpts.mkStr "require('ts-rainbow').strategy.global"
+        "The query to use for finding delimiters.";
```

Comment:
Mention in the documentaiton that this should be lua code

Reply:
Done